client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -252,6 +253,7 @@ export class PlanFeatures2023Grid extends Component<
 > {
 	state = {
 		showPlansComparisonGrid: false,
+		isSelectedPlan: '',
```

Comment:
`Non-blocking / Nitpick:`

Could we switch this from `isSelectedPlan` to `selectedPlan`. The `is` prefix implies a boolean datatype

Reply:
Sounds good, I will update it